### PR TITLE
[ko] Update outdated files in `dev-1.24-ko.1` (M139-M162)

### PR DESCRIPTION
**Pull request by jihoon-seo**
한 PR에 너무 많은 변경사항을 포함시키면 리뷰하기 어려울 수 있어서, 적절히 분할하여 올립니다. 😊

- Related issue: #33472 (**dev-1.24-ko.1**)

> ### 24 files to be modified
> * [ ]  M139.  `content/en/examples/admin/logging/two-files-counter-pod-agent-sidecar.yaml`  | 1(+XS) 1(-)
> * [ ]  M140.  `content/en/examples/admin/logging/two-files-counter-pod-streaming-sidecar.yaml`  | 5(+XS) 5(-)
> * [ ]  M141.  `content/en/examples/admin/logging/two-files-counter-pod.yaml`  | 1(+XS) 1(-)
> * [ ]  M142.  `content/en/examples/admin/resource/limit-range-pod-1.yaml`  | 4(+XS) 4(-)
> * [ ]  M143.  `content/en/examples/admin/resource/limit-range-pod-2.yaml`  | 4(+XS) 4(-)
> * [ ]  M144.  `content/en/examples/admin/resource/limit-range-pod-3.yaml`  | 1(+XS) 1(-)
> * [ ]  M145.  `content/en/examples/application/job/cronjob.yaml`  | 1(+XS) 1(-)
> * [ ]  M146.  `content/en/examples/application/job/job-tmpl.yaml`  | 1(+XS) 1(-)
> * [ ]  M147.  `content/en/examples/application/mysql/mysql-configmap.yaml`  | 2(+XS) 0(-)
> * [ ]  M148.  `content/en/examples/application/mysql/mysql-services.yaml`  | 1(+XS) 0(-)
> * [ ]  M149.  `content/en/examples/controllers/daemonset.yaml`  | 5(+XS) 2(-)
> * [ ]  M150.  `content/en/examples/controllers/fluentd-daemonset-update.yaml`  | 6(+XS) 2(-)
> * [ ]  M151.  `content/en/examples/controllers/fluentd-daemonset.yaml`  | 6(+XS) 2(-)
> * [ ]  M152.  `content/en/examples/debug/counter-pod.yaml`  | 1(+XS) 1(-)
> * [ ]  M153.  `content/en/examples/pods/init-containers.yaml`  | 1(+XS) 1(-)
> * [ ]  M154.  `content/en/examples/pods/inject/dependent-envars.yaml`  | 1(+XS) 1(-)
> * [ ]  M155.  `content/en/examples/pods/pod-with-node-affinity.yaml`  | 2(+XS) 3(-)
> * [ ]  M156.  `content/en/examples/pods/resource/memory-request-limit-3.yaml`  | 2(+XS) 2(-)
> * [ ]  M157.  `content/en/examples/pods/resource/memory-request-limit.yaml`  | 2(+XS) 2(-)
> * [ ]  M158.  `content/en/examples/pods/security/hello-apparmor.yaml`  | 1(+XS) 1(-)
> * [ ]  M159.  `content/en/examples/policy/restricted-psp.yaml`  | 1(+XS) 0(-)
> * [ ]  M160.  `content/en/examples/service/networking/hostaliases-pod.yaml`  | 1(+XS) 1(-)
> * [ ]  M161.  `content/en/includes/default-storage-class-prereqs.md`  | 2(+XS) 2(-)
> * [ ]  M162.  `content/en/releases/_index.md`  | 2(+XS) 2(-)

[참고]
변경 사항 중에서 링크 업데이트를 반영하지 않은 부분이 있는데,
이는
- #33781 에서 처리될 예정이거나,
- #33472 의 R1-R9 를 처리하는 PR에서 
  링크도 함께 업데이트 하거나,
- #33472 가 마무리될 쯤에 
  `dev-1.24-ko.1` 의 링크들을 다시 한 번 점검하고 업데이트 하는 것이 좋을 것 같습니다.

/language ko

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @seokho-son

File: content/ko/examples/controllers/daemonset.yaml [hunk 1]
```diff
@@ -15,8 +15,11 @@ spec:
         name: fluentd-elasticsearch
     spec:
       tolerations:
-      # this toleration is to have the daemonset runnable on master nodes
-      # remove it if your masters can't run pods
+      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다
+      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다
```
Comment (seokho-son) [inline content/ko/examples/controllers/daemonset.yaml:19]:
```suggestion
      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다.
      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다.
```

File: content/ko/examples/controllers/fluentd-daemonset-update.yaml [hunk 2]
```diff
@@ -19,9 +19,13 @@ spec:
         name: fluentd-elasticsearch
     spec:
       tolerations:
-      # 이 톨러레이션(toleration)은 마스터 노드에서 실행 가능한 데몬셋이
-      # 마스터에서 파드를 실행할 수 없는 경우 이를 제거하는 것이다
+      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다
+      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다
```
Comment (seokho-son) [inline content/ko/examples/controllers/fluentd-daemonset-update.yaml:23]:
```suggestion
      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다.
      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다.
```

File: content/ko/examples/controllers/fluentd-daemonset.yaml [hunk 3]
```diff
@@ -19,9 +19,13 @@ spec:
         name: fluentd-elasticsearch
     spec:
       tolerations:
-      # 이 톨러레이션(toleration)은 마스터 노드에서 실행 가능한 데몬셋이
-      # 마스터에서 파드를 실행할 수 없는 경우 이를 제거하는 것이다
+      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다
+      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다
```
Comment (seokho-son) [inline content/ko/examples/controllers/fluentd-daemonset.yaml:23]:
```suggestion
      # 이 톨러레이션(toleration)은 데몬셋이 컨트롤 플레인 노드에서 실행될 수 있도록 만든다.
      # 컨트롤 플레인 노드가 이 파드를 실행해서는 안 되는 경우, 이 톨러레이션을 제거한다.
```

**Review by seokho-son — COMMENTED** [review]
Minor things :)

**Comment by jihoon-seo** [comment]
@seokho-son 리뷰 감사합니다. 해당 사항을 반영하고 squash까지 하였습니다. 확인 부탁드립니다. 😊

**Pull request merged**